airflow/operators/python.py

```diff
@@ -258,6 +258,12 @@ def task(
     :type multiple_outputs: bool
 
     """
+    # try to infer from  type annotation
+    if python_callable and multiple_outputs is None:
+        sig = signature(python_callable).return_annotation
+        ttype = getattr(sig, "__origin__", None)
+
+        multiple_outputs = sig != inspect.Signature.empty and ttype in (dict, Dict)
```

Comment:
Do we not support list too?

Reply:
Infact, yes, as given by the example in the original issue:

```python
@task
def my_function() -> Tuple[int, int, int]:
    return 1, 2, 3
```

Reply:
We decided not to automatically unroll tuples/lists: https://github.com/apache/airflow/pull/8962#discussion_r429146569

Reply:
Ahahaaaa. By "we" you actually mean me 😁